utils/mgmt_system/kubernetes.py

```diff
@@ -0,0 +1,208 @@
+import os
+from conf import cfme_data, credentials
+from utils.rest_client import ContainerClient
+from collections import namedtuple
+
+"""
+Related yaml structures:
+
+[cfme_data]
+container_managers:
```

Comment:
yup, sure, I was wondering about that as well - putting it under management_systems will make them separable only by "type"